runtime/buffer.cc

```diff
@@ -759,6 +759,7 @@ SLINKY_NO_STACK_PROTECTOR SLINKY_ALWAYS_INLINE inline void for_each_impl(span<co
 
     // We need to replace the implementation of the last loop.
     for_each_loop<BufsSize>* inner_loop = offset_bytes_non_null(loop, -sizeof_for_each_loop(bufs.size()));
+    assert(inner_impl);
     (void)inner_impl;
```

Comment:
Why doesn't this solve the problem?

Reply:
Doesn't this fix unused, bit not unitialized?